### ISSUE TITLE
[Bug] templ version check failed

### ISSUE THREAD

**Issue by vadhe** [Melkeydev/go-blueprint#247]
### What is the problem?

templ version check failed: failed to parse go.mod 

### Operating System

macOS ventura

### Architecture Version (x86, x64, arm, etc)

x86

### Steps to reproduce

make watch
### Relevant log output

```shell
(!) templ version check failed: failed to parse go.mod file: /Users/aldyfadhe/Desktop/side-projects/movie-tracker/go.mod:3: invalid go version '1.22.3': must match format 1.23
```

**Comment by Ujstor** [comment]
This is a pretty strange error. I am seeing it for the first time. Not sure what is causing this.

I found an issue in the templ repository that is related to this:
https://github.com/a-h/templ/issues/355

Try manually setting the version in go.mod to 1.22 and see what happens. It's the only thing that comes to mind.

**Comment by vadhe** [comment]
I'm trying to update my Go version to the current version go 1.22.3 but the error still happens

**Comment by vadhe** [comment]
it's solved, I changed version 1.22.3 to 1.22 and updated my templ version
